libcxx/test/std/thread/thread.mutex/thread.lock/thread.lock.guard/mutex.verify.cpp

```diff
@@ -16,10 +14,12 @@
 
 #include <mutex>
 
+struct MyMutex {};
```

Comment:
I think you need to use a valid `BasicLockable` all the time, otherwise the code is ill-formed. I suggest you define `MyMutex` in a local header like `types.h` and include it from all the tests in this directory that use `MyMutex`.

Reply:
Done.